src/components/Main-nav/MainNav.scss

```diff
@@ -0,0 +1,266 @@
+@import '../shared/main.scss';
+
+.main-menu{
+	&__nav{
+		width: 100%;
+		height: 5vh;
+		display: flex;
+		cursor: pointer;
+		font-family: $font-family;
+
+		@include large{
+			width: 100%;
+			height: 100px;
+			position: relative;
+			//background: #f9c833;
+			background: #c21500;  /* fallback for old browsers */
+			background: -webkit-linear-gradient(left, #ffc500, #c21500);  /* Chrome 10-25, Safari 5.1-6 */
```

Comment:
We don't need to use fallbacks in the code level. If you need them, use them in a build level (Webpack).